### ISSUE TITLE
Reply header becomes "HeaderChangeQuote"

### ISSUE THREAD

**Issue by kNishatani** [caligraf/ChangeQuote#43]
I am using Thunderbird 115.6.1 and changeQuote 1.7.11 on Windows 11 in a Japanese environment.
When I create a reply to an email, only the following may appear in the reply header
`************HeaderChangeQuote*****************`

Currently, we have confirmed that this occurs under one of the following conditions
- When creating a reply to some emails, regardless of the email account (the condition of "some emails" has not been specified)
- When creating a reply with changeQuote disabled

**Comment by caligraf** [comment]
To fix the problem you can reset the values of the followings settings:
mailnews.reply_header_originalmessage
mailnews.reply_header_ondateauthorwrote
mailnews.reply_header_authorwrotesingle
mailnews.reply_header_authorwroteondate
mailnews.reply_header_type

1. go to settings available from tools menu:

![image](https://github.com/caligraf/ChangeQuote/assets/44136069/34cea448-c222-4c5c-bb19-a9052469197a)

2. click on the button Config Editor at the bottom of settings tab:

![image](https://github.com/caligraf/ChangeQuote/assets/44136069/0c13c9b0-3e30-4a87-8437-58c0745b5e75)

3. Reset settings values that are in bold by clicking the reset on right on each line:

![image](https://github.com/caligraf/ChangeQuote/assets/44136069/88c27310-0eec-4f0c-b163-6d43c919fd5b)
These are the values you should have:
![image](https://github.com/caligraf/ChangeQuote/assets/44136069/febc1991-3dd9-4a17-964c-77eda6d4c7c7)

The previous versions of changeQuote were using settings values, but it is not the case anymore. There is a step at start of the extension that migrate the settings into local storage and reset the values to default.

**Comment by kNishatani** [comment]
HeaderChangeQuote is never displayed anymore and it is back to the normal "On %Y/%m/%d %H:%i, %%6 wrote:".
`mailnews.reply_header_originalmessage` entry seems to be the cause. Thank you very much.

The second condition is now resolved, but the first condition is not resolved because changeQuote is not working as expected.
We have confirmed that changeQuote is not working in creating a reply to an email from a specific address.
All emails from that address are auto-sent emails, and it can be considered that the structure of the email is affecting the behavior of changeQuote.
What should we report in this case?

Also, should we submit a new issue for this case as "changeQuote does not work with some email replies"?
